apps/ex_wire/lib/ex_wire/p2p/connection.ex

```diff
@@ -22,7 +22,8 @@ defmodule ExWire.P2P.Connection do
           sent_message_count: integer(),
           datas: [binary()],
           last_error: any() | nil,
-          is_outbound: boolean()
+          is_outbound: boolean(),
+          timer: Time.t() | nil
```

Comment:
This is not really a timer, instead a time. Might be better to name it `created_time`, or something else based on how it's used.